lib/slack/onEvent/message.js

```diff
@@ -193,5 +199,5 @@ module.exports = function (Doorman) {
     }
   }
 
-  Doorman.Slack.on(Doorman.Slack.rtm_events.MESSAGE, msg => checkMessageForCommand(msg, false));
+  Doorman.Slack.on('message', msg => checkMessageForCommand(msg, false));
```

Comment:
The new Slack SDK has changed the location of these helpers, and I feel it would be better to know _exactly_ what message type is being handled —  no need to configure these anyway, so it is better to be explicit.